readme.md

```diff
@@ -103,6 +107,8 @@ Once installation is working, take a look at the existing code to make sure you
   _NOTE: We will not go into this too deeply for the sake of brevity, however this is a really interesting topic! I would encourage you to look into this more on your own, if you're interested._
 
 * **Your Answer:**
+Saltrounds are how many rounds it goes through to give you a secure hash.  Increasing the salt rounds increases the amount of time required to cause a delay if someone is trying to guess / brute force in.  
+
```

Comment:
Saltrounds refers to the Cost to the server to create a secure hash. Increasing SaltRounds requires more time and resources on the server for each hash request, but creates a hash that is more difficult to brute force.